File: src/01-easy/the-descent/solution.dart
import 'dart:io';
void main() {
    while (true) {
        int max = -1, idx = 0;
        for (int i = 0; i < 8; i++) {
            int h = int.parse(stdin.readLineSync()!);
            if (h > max) { max = h; idx = i; }
        }
        print(idx);
    }
}

File: src/01-easy/the-descent/solution.go
package main
import "fmt"
func main() {
    for {
        max, idx := -1, 0
        for i := 0; i < 8; i++ {
            var h int
            fmt.Scan(&h)
            if h > max { max = h; idx = i }
        }
        fmt.Println(idx)
    }
}
